Update validator FAQ (#495)
lukso-network/docs: docs/faq/lukso.md

Please join our [Discord](https://discord.gg/lukso) and get in touch if you have terrific ideas for the new digital lifestyle economy!

## Why are LSPs not proposed as ERCs?

The LSPs are part of a larger set of core standards that are all interoperable and cohesive together. Nonetheless, they do function equally for any EVM chain, the ability to make these standards understood requires a less noisy and neutral place, than the current ERC space.

Therefore, we deem a separate repo **[(LIPs/LSPs)](https://github.com/lukso-network/LIPs)** as necessary for the success and transparency of these new sets of standards, many of which use ERCs (ERC165, ERC173, ERC725) as a core basis.

## I found a bug. Where can I report it?

> You can create a GitHub issue in the related project's repository. If you can't do it, you can send us a message on our [Discord](https://discord.gg/lukso) server.

Reviewer: ```suggestion
You can create a GitHub issue in the related project's repository. Feel free to message us on our [Discord](https://discord.gg/lukso) server if you're not experienced with GitHub.
```
